### PR TITLE
Travis support

### PR DESCRIPTION
**Pull request by dotnetCarpenter**
You might need to login to travis before this works

### CODE REVIEW

**Comment by dionyziz** [comment]
Flicked on the travis switch for this repo. Let's merge this after #19 #20 are merged.

**Comment by dotnetCarpenter** [comment]
All of my PR's are cumulative so you can just merge this one and discard the others. I only split them up so that it wasn't too many changes you had to consider. But if you are fine with all of them, then just merge this.
But let me change the npm name to `stream.js` (since [stream] is already taken) and fix the failing test case and push to this branch, first though.

**Comment by dotnetCarpenter** [comment]
It turns out that [stream](https://www.npmjs.com/package/stream) and [stream.js](https://www.npmjs.com/package/stream.js) is already taken. stream has no active development but has lots of dependents. steam.js 1.0.1 was just released 2 months ago but has zero dependents. We could ask but it doesn't seem likely that they would give us their npm name. Got a name suggestion? `list` is also taken but it's empty. It's own by etechi <cy2000@gmail.com>

**Comment by dotnetCarpenter** [comment]
Turns out that travis can't really test node v0.4.* should we up the node engine requirement in package.json?

**Comment by dionyziz** [comment]
steam.js or stream.js?

**Comment by dotnetCarpenter** [comment]
both are taken :-1: [steam.js](https://github.com/JulioC/steam.js#readme)

**Comment by dotnetCarpenter** [comment]
maybe we just don't publish to npm and keep stream.js a secret?

btw thank you so much for this lib. I finally wrapped my head around streams

**Comment by dionyziz** [comment]
Glad you liked it, thank you so much for your help in fixing these basic packaging issues :)

**Comment by dionyziz** [comment]
I think we can keep it secret. Or we can use 'Stream' in upper-case? I tried "stream", "streams", "stream.js", "streamjs". Another option is "streams.js" but this changes the name of the library.

**Comment by dotnetCarpenter** [comment]
npm says `1. name can no longer contain capital letters`. I'm checking names with `npm home [name]`

**Comment by dotnetCarpenter** [comment]
Lets keep it a secret. You can still publish it scoped to dionyziz

**Comment by dionyziz** [comment]
sounds good

File: package.json [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
+    {
+      "name": "Ryan Scheel",
+      "email": "ryan.havvy@gmail.com",
+      "web": "http://widget.mibbit.com/?channel=%23havvy&server=irc.mibbit.net"
+    },
+    {
+      "name": "Dionysis Zindros",
+      "email": "dionyziz@gmail.com",
+      "web": "http://dionyziz.com/"
+    }
```
Comment (dionyziz) [inline package.json]: Feel free to add yourself also

**Comment by dotnetCarpenter** [comment]
How about code coverage? [Codecov](https://codecov.io/) shows more data than [Coveralls](https://coveralls.io/)

**Comment by dionyziz** [comment]
Sounds good, can you add support for this?

**Comment by dotnetCarpenter** [comment]
I think so. I haven't done it with jasmine but pretty sure there is a module out there

**Comment by dotnetCarpenter** [comment]
You should just merge this in. I will do another PR with the coverage stuff

**Pull request merged**